sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/ImplementISerializableCorrectly.cs

```diff
@@ -95,17 +95,27 @@ protected Serializable_NoAttribute_1(SerializationInfo info, StreamingContext co
 
     [Serializable]
     public class Serializable_ExplicitImplementation : ISerializable
-//               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Noncompliant
     {
         public Serializable_ExplicitImplementation()
         { /*do something*/ }
         protected Serializable_ExplicitImplementation(SerializationInfo info, StreamingContext context)
         { /*do something*/ }
         void ISerializable.GetObjectData(SerializationInfo info, StreamingContext context)
```

Comment:
Please add a comment stating this is a FN, as the virtual method is not called from implicit implementation.

Reply:
done